geode-core/src/main/java/com/gemstone/gemfire/management/internal/ManagementAgent.java

```diff
@@ -509,8 +510,7 @@ private void registerAccessControlMBean() {
 
 
   private boolean isIntegratedSecurity() {
```

Comment:
We can get rid of this method. I believe this is called to check if we need to enabled pulse security in the same file as well. Yeah, what about pulse security? Do we need to add another enum or just say if jmx is enabled. pulse should be enabled as well?